platform/xsight/xsight-sai.dep

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
It probably would make more sense splitting this PR to a platform one and then add the specific device. It's a pretty large PR, hard for someone to actually review all.

Reply:
There are actually 2 new platforms in this PR: 9632xx and 9632xq. (accton platforms)
Each platform has several SKUs supported. 

The device is a from a new vendor called xsight, So I assume that if we want to reduce the size of the PR, we can separate it to a the new device (xsight) and a single platform (e.g 9632xq) and then add the 2nd platform. (9632xx)

It will still create a pretty big first PR. 
Do you have any better suggestions?